src/library_html5_webgl.js

```diff
@@ -641,6 +638,10 @@ function handleWebGLProxying(funcs) {
 }
 
 handleWebGLProxying(LibraryHtml5WebGL);
+#endif // USE_PTHREADS
+
+#if LibraryManager.has('library_webgl.js')
+autoAddDeps(LibraryHtml5WebGL, '$GL');
```

Comment:
Should the comment on lines 74-78 be moved here? And, why is this change needed in this PR?

Reply:
Mostly for the deps_info test itself.    But actually I found it isn't actually exercised anymore so I can revert/split out this part of the change.